cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/repositories.proto

```diff
@@ -85,6 +85,9 @@ message AddPackageRepositoryRequest {
   string url = 6;
 
   // The interval at which to check the upstream for updates (in time+unit)
+  // Some plugins may, addionally, support other formats, for instance,
+  // a cron expression.
+  // e.g., "*/10 * * * *" will be equivalent to "10m"
   // Optional. Defaults to 10m if not specified
   string interval = 7;
```

Comment:
I've added this note to document the fact that other plugins might support other formats while also supporting the "time+unit" one.